### PR TITLE
Set lockbot reason 🔒🤖

### PR DESCRIPTION
**Pull request by batpigandme**
* Stop lockbot from marking issues as resolved. 
* Add message explaining what lockbot does.

* See https://github.com/dessant/lock-threads/issues/7 re. new `setLockReason` param.

### CODE REVIEW

File: .github/lock.yml [hunk 1]
```diff
@@ -12,6 +12,15 @@ lockLabel: false
 # Comment to post before locking. Set to `false` to disable
 lockComment: false
 
+# Stop lockbot from closing as "resolved"
+setLockReason: false
+
+# Comment to post before locking. Set to `false` to disable
+lockComment: >
+  This thread has been automatically locked as there has not been
```
Comment (hadley) [inline .github/lock.yml]: How about:

This old issue has been automatically locked. If you believe you have found a related problem, please file a new issue (with reprex) and link to this issue.

**Comment by batpigandme** [comment]
@hadley reworded as requested ✔️

**Comment by batpigandme** [comment]
I can add the link to reprex at the end there.

File: .github/lock.yml [hunk 2]
```diff
@@ -12,6 +12,15 @@ lockLabel: false
 # Comment to post before locking. Set to `false` to disable
 lockComment: false
```
Comment (dessant) [inline .github/lock.yml]: Duplicate keys will trip up the yaml parser.

Comment (batpigandme) [reply .github/lock.yml]: Whoops! Thanks @dessant!

**Comment by krlmlr** [comment]
Can we merge this?

**Comment by batpigandme** [comment]
As long as the appveyor failure doesn't matter ✅

**Comment by krlmlr** [comment]
Thanks! I don't think these failures are related.

**Pull request merged**